src/Pages/SingleArticle.jsx

```diff
@@ -7,10 +7,16 @@ export const SingleArticle = () => {
     const [singleArticle, setSingleArticle] = useState([]);
     const [isLoading, setIsLoading] = useState(false)
     const [count, setCount] = useState(0);
+    const [err, setErr] = useState(null);
     const { article_id } = useParams();
 
     const changeVote = () => {
+        setErr(null)
         updateVote(article_id)
+        .catch((err) => {
+            setCount((count) => count - 1);
+            setErr('Something went wrong, please try again.')
+        })
```

Comment:
I would jsut move this to above the request

Reply:
So just above the updateVote that links to the api request?